Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/UserPreferences.java

```diff
@@ -142,6 +199,86 @@ public static void SavePrefsToIni(Context ctx)
 		// Current CPU core being used. Falls back to interpreter upon error.
 		String currentEmuCore = prefs.getString("cpuCorePref", "0");
 
+		// Current GC analog range setup. Falls back to 100.000000 (90 for triggers) upon error.
+		String MainRadius_0 = prefs.getString("MainRadius_0", "100.000000");
```

Comment:
Like I previously stated. These variable names should be in the form `mainRadius0`, which is normal Java naming conventions.